### ISSUE TITLE
<ul> tags cannot contain <p> tags

### ISSUE THREAD

**Issue by lizgarciao** [pulibrary/audit-Liz#7]
### URL
https://lizgarciao.github.io/

### Issues
ul tags cannot contain p tags

### Severity
High

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Screen reader can have difficulty reading the list.

### Recommended fix
Remove p tags

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Notes
 https://www.w3.org/TR/WCAG20-TECHS/H48.html#:~:text=technique%20relates%20to%3A-,Success%20Criterion%201.3.1%20(Info%20and%20Relationships),-How%20to%20Meet
